app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :to, :from, :returned_at
+  belongs_to :user
+  belongs_to :book
+  def returned_at
+    object.to > Time.zone.today ? null : object.to
```

Comment:
In Ruby we use nil instead of null, i didn't knew that null existed as a keyword in ruby haha